test/benchee/formatters/console_test.exs

```diff
@@ -21,6 +21,31 @@ defmodule Benchee.Formatters.ConsoleTest do
     assert Regex.match?(~r/Third/,  result_3)
   end
 
+  test "adjusts the label width to longest name" do
+    third_name = String.duplicate("a", 40)
+    first  = {"First",  %{average: 100.0, ips: 10_000.0,
+                          std_dev_ratio: 0.1, median: 90.0}}
+    second = {"Second", %{average: 200.0, ips: 5_000.0,
+                          std_dev_ratio: 0.1, median: 195.5}}
+    third  = {third_name,  %{average: 400.0, ips: 2_500.0,
+                             std_dev_ratio: 0.1, median: 375.0}}
+
+    # Just normally long names, expect default width of 30 + 13
```

Comment:
kept wondering where the 13 came from... it's the width of the `ips` cause it's right aligned, isn't it? In that case it'd be cool if we defined it as an attribute and then subtracted it at the end for the `assert_column_width` method so that we could assert against the actual width of the name column (that doesn't go into the ips column) e.g. 30 and 41 in this test